exercises/ex2.cpp

```diff
@@ -5,47 +5,20 @@
 */
 
 #include <iostream>
+#include <string.h>
 using namespace std;
 
 int main()
 {
   int week;
-
+  string arr[7]={"Monday","Tuesday","Wednesday","Thursday","Friday","Saturday","Sunday"};
   cout << "Enter week number(1-7): " << endl;
   cin >> week;
 
-  if (week == 1)
-  {
-    cout << "Monday" << endl;
-  }
-  else if (week == 2)
-  {
-    cout << "Tuesday" << endl;
-  }
-  else if (week == 3)
-  {
-    cout << "Wednesday" << endl;
-  }
-  else if (week == 4)
-  {
-    cout << "Thursday" << endl;
-  }
-  else if (week == 5)
-  {
-    cout << "Friday" << endl;
-  }
-  else if (week == 6)
-  {
-    cout << "Saturday" << endl;
-  }
-  else if (week == 7)
-  {
-    cout << "Sunday" << endl;
-  }
-  else
-  {
-    cout << "Invalid input! Please enter week number between 1-7." << endl;
-  }
+  if(week >0 && week < 8 )
+    cout<<arr[week-1];
+  else 
+    cout << "Invalid input! Please enter week number between 1-7." << endl; 
 
   return 0;
```

Comment:
Potresti rendere più esplicito il valore di ritorno del **main** aggiungendo in cima al codice, subito dopo gli "include"

```c++
#define EXIT_SUCCESS 0
```

e alla fine del **main**, al posto di "return 0;"

```c++
return EXIT_SUCCESS;
```